src/main/webapp/find-events.html

```diff
@@ -125,23 +125,26 @@ <h1 id="modal-title" tabindex="0">Create a Post</h1>
           <span class="modal-span">
             <p id="modal-time-text" tabindex="0">end:</p>
             <input aria-label="end hour" id="modal-end-hour" max="12" maxlength="2" min="1" name="endHour" placeholder="h" step="1" tabindex="0" type="number">
-            <input aria-label="end minute" id="modal-end-minute" max="60" maxlength="2" min="0" name="endMinute" placeholder="m" step="1" tabindex="0" type="number">
+            <input aria-label="end minute" id="modal-end-minute" max="59" maxlength="2" min="0" name="endMinute" placeholder="m" step="1" tabindex="0" type="number">
             <select aria-label="end AM or PM?" id="end-am-or-pm" name="endAMorPM" role="listbox">
               <option aria-hidden="false" aria-label="am" role="option" value="am">am</option>
               <option aria-hidden="false" aria-label="pm" role="option" value="pm">pm</option>
             </select>
           </span>
 
-          <input aria-label="location" id="modal-location" maxlength="100" name="location" placeholder="location" tabindex="0" type="text" required><br>
-          <input aria-label="type of food" id="modal-type-food" maxlength="75" name="typeOfFood"  placeholder="type of food" tabindex="0" type="text" required><br>
-          <input aria-label="how many people can it feed"  id="modal-num-people" maxlength="50" name="numberOfPeopleItFeeds" placeholder="how many people can it feed?" tabindex="0" type="text" required><br>
-          <textarea aria-label="description" id="modal-description" maxlength="500" name="description" placeholder="description" rows="3" tabindex="0" ></textarea><br>
+          <span id="modal-date-tip" class="modal-span" aria-live="polite"></span>
+
+          <input aria-label="location" id="modal-location" maxlength="100" name="location" placeholder="location" tabindex="0" type="text" onpaste="return false" required><br>
+          <input aria-label="type of food" id="modal-type-food" maxlength="75" name="typeOfFood"  placeholder="type of food" tabindex="0" type="text" onpaste="return false" required><br>
+          <input aria-label="how many people can it feed"  id="modal-num-people" maxlength="3" min="1" max="999" name="numberOfPeopleItFeeds" placeholder="how many people can it feed?" tabindex="0" type="number" required><br>
+          <textarea aria-label="description" id="modal-description" maxlength="500" name="description" placeholder="description" rows="3" tabindex="0" onpaste="return false"></textarea><br>
```

Comment:
Should we also put an minimum length for this?

Reply:
I suppose we could, although personally I am open to letting people write really short descriptions - they're already not allowed to leave it blank (which is checked with JS I believe)

Reply:
Yeah! I'm also open to letting people write short descriptions since as an club, you would want to get out the post as soon as possible! Maybe like at least 15 characters?